src/arrays/arraymath.jl

```diff
@@ -109,7 +109,7 @@ Base.scale(qarr::AbstractQuArray, num::Number) = scale(num, qarr)
 *(num::Number, qarr::AbstractQuArray) = scale(num, qarr)
 *(qarr::AbstractQuArray, num::Number) = scale(qarr, num)
 /(qarr::AbstractQuArray, num::Number) = scale(1/num, qarr)
-
+\(qarr::AbstractQuArray, v::AbstractQuArray) = QuArray(\(coeffs(qarr), coeffs(v)))
```

Comment:
You should determine the return type using `similar_type` (see +,-,* for example). It's probably better to restrict the arguments to 2 `AbstractQuMatrix` and `AbstractQuMatrix` and `AbstractQuVector` (so you need two separate functions). Anything else won't work.